modules/govuk_jenkins/files/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy

```diff
@@ -26,6 +26,16 @@ def checkoutFromGitHubWithSSH(String repository, String org = 'alphagov', String
 }
 
 /**
+  * Check if the git HEAD is ahead of master.
+  * This will be false for development branches and true for release branches,
+  * and master itself.
+  */
+def isCurrentCommitOnMaster = sh(
```

Comment:
This looks like a variable assignment rather than a function definition, which I think means it will be evaluated when this helper script is loaded, and therefore will be run _before_ `git clone`. That seems wrong, but I might have just misunderstood what's going on.